CHANGES.rst

```diff
@@ -53,6 +53,9 @@ New Features
   - The ``spectral_density`` equivalency now supports transformations of
     luminosity density. [#5151]
 
+  - ``Quantity`` now accepts ``Unit`` objects and strings that can be converted
```

Comment:
Should this maybe be "...and strings that can be parsed by the ``Unit`` constructor."